cmd/process-agent/collector_api_test.go

```diff
@@ -116,6 +116,41 @@ func TestSendProcMessage(t *testing.T) {
 	})
 }
 
+func TestSendProcessDiscoveryMessage(t *testing.T) {
```

Comment:
nit: Can you add a description for this test in a comment?

Reply:
I think the test name is already pretty descriptive and it follows the test conventions laid out by the rest of the files.